packages/extensions/core/resources/plugin-csharp.md

```diff
@@ -2,17 +2,25 @@
 
 The V2 version of the C# Generator.
 
-``` yaml $(csharp) && $(preview) && !isRequested('@autorest/csharp')
-# default the v2 generator to using the last stable @microsoft.azure/autorest-core 
+``` yaml $(csharp) && !$(legacy) && !$(v2) && !isRequested('@microsoft.azure/autorest.csharp')
+version: ~3.0.6298
```

Comment:
Where did you get `3.0.6298` version? We should use [@autorest/csharp@3.0.0-beta.20210205.1](https://www.npmjs.com/package/@autorest/csharp/v/3.0.0-beta.20210205.1) version or above.